### PR TITLE
[JENKINS-68448] Prepare Java 11 requirement

### PR DESCRIPTION
**Pull request by alecharp**
Explicitely declares JAXB plugin as a dependency.

See [JENKINS-68448](https://issues.jenkins.io/browse/JENKINS-68448).

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -87,6 +87,11 @@
       <artifactId>plugin-util-api</artifactId>
       <version>${plugin-util-api.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>jaxb</artifactId>
+      <version>2.3.6-1</version>
```
Comment (uhafner) [inline pom.xml:93]: Is jaxb not in the BOM?

**Comment by uhafner** [comment]
Can you rebase and retry? I updated the parent pom to the latest version now.

File: pom.xml (hunk 1 above)
Comment (alecharp) [reply pom.xml:93]: it is, you are absolutely right.

**Review by uhafner — APPROVED** [review]

**Comment by uhafner** [comment]
Thanks!

**Pull request merged**